.github/ISSUE_TEMPLATE/deprecation.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Deprecation
+about: For the Octopus team to make customers aware of upcoming deprecations.
+title: ''
+labels: tag/breaking-change
+assignees: ''
+
+---
+
+**_Are you a customer of Octopus Deploy? Please contact [our support team](https://octopus.com/support) if you have any questions regarding an upcoming deprecation.**
+
+# Prerequisites
+
+- [ ] I have updated the [public deprecation schedule](https://octopus.com/docs/deprecations)
```

Comment:
I like the idea of having this as a checkbox *somewhere*, but I don't think that document would be updated at the time this issue is created (assuming we're creating it at the same time as the internal RFC)

Reply:
I don't think it makes a huge difference which one comes first, as long as the public notice is up by the time the issue is closed. No strong opinions here, so open to moving it somewhere else.